packages/main/src/components/Toolbar/index.tsx

```diff
@@ -265,21 +265,29 @@ const Toolbar = forwardRef((props: ToolbarPropTypes, ref: Ref<HTMLDivElement>) =
     [onClick, active]
   );
 
+  const prevChildren = useRef(React.Children.toArray(children));
+  const debouncedOverflowChange = useRef(debounce(onOverflowChange, 60)).current;
```

Comment:
This implies that the `onOverflowChange` prop never changes, IMO we would need an effect to update the debounced function in case the `onOverflowChange` prop is updated.

Reply:
Good point. Fixed